app/helpers/kaui/date_helper.rb

```diff
@@ -13,7 +13,7 @@ def format_date(date, timezone="Pacific Time (US & Canada)")
 
       # If not, convert into account timezone and return the date part only
       parsed_date = DateTime.parse(date.to_s).in_time_zone(timezone)
-      parsed_date.to_s(:date_only)
+      parsed_date.to_s(:pretty)
```

Comment:
`format_date` must return a date (i.e. `LocalDate` in Java), see comment line 14. This would break lots of screens.